content/blog/2016-05-09-when-to-use-varbit-chunks.md

```diff
@@ -0,0 +1,159 @@
+---
+title: When (not) to use varbit chunks
+created_at: 2016-05-09
```

Comment:
I think both dating and posting it the day after tomorrow probably best. I don't think that many people go to `/blog` directly, so I'm not too worried about it overshadowing the CNCF post.